src/Notes/NavigationNudge.php

```diff
@@ -33,13 +33,38 @@ public function __construct() {
 		add_action( 'update_option_' . Navigation::TOGGLE_OPTION_NAME, array( $this, 'action_note' ), 10, 2 );
 	}
 
+	/**
+	 * Should this note exist? (The navigation feature should exist.)
+	 */
+	public static function should_note_exist() {
+		return Features::exists( 'navigation' );
+	}
+
+	/**
+	 * Delete this note if the navigation feature does not exist, unless the note has been soft-deleted already.
+	 */
+	public static function delete_if_not_supported() {
+		if ( ! self::should_note_exist() ) {
+			$data_store = Notes::load_data_store();
+			$note_ids   = $data_store->get_notes_with_name( self::NOTE_NAME );
+
+			if ( ! empty( $note_ids ) ) {
+				$note = Notes::get_note( $note_ids[0] );
+
+				if ( ! $note->get_is_deleted() ) {
```

Comment:
I think we should also check if the note is actioned here as well.  If a note is actioned, we should not resurrect it.

Reply:
Good catch! I will make that change.